### ISSUE TITLE
can you please upgrade it to asp.net core 2.0 and angular 5?

### ISSUE THREAD

**Issue by eddyzhu1980** [mosh-hamedani/vega#9]
can you please upgrade it to asp.net core 2.0 and angular 5?

**Comment by caweb** [comment]
would be great! please

**Comment by MarkPhillips** [comment]
Yes please.  The versions being spoken of early in the video do not inspire confidence that taking this course is the best use of my time.
